<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>movie_window</class>
 <widget class="QWidget" name="movie_window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>731</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Details</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="2">
        <widget class="QLabel" name="movieYear_label">
         <property name="text">
          <string>Year</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLabel" name="movieLengthIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" rowspan="3">
        <widget class="QLabel" name="movieCover_label">
         <property name="text">
          <string>Cover</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="movieYearIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="movieGenreIcon_label">
         <property name="text">
          <string/>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="movieLength_label">
         <property name="text">
          <string>Length</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="movieGenre_label">
         <property name="text">
          <string>Genre</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="3">
        <widget class="QLabel" name="movieDescription_label">
         <property name="text">
          <string>Description</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="3">
        <widget class="QLabel" name="movieTitle_label">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Edit/Delete</string>
      </attribute>
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0" colspan="2">
        <widget class="QLabel" name="editMovie_label">
         <property name="text">
          <string>Edit movie</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="movieTitle_label_2">
         <property name="text">
          <string>Title:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="movieTitle_lineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="movieLength_label_2">
         <property name="text">
          <string>Length:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="hour_label">
           <property name="text">
            <string>Hour</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="hour_spinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="maximum">
            <number>100</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Preferred</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="minutes_label">
           <property name="text">
            <string>Minutes</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="minutes_spinBox">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="movieYear_label_2">
         <property name="text">
          <string>Year:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="movieYear_comboBox"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="filmDirector_label">
         <property name="text">
          <string>Film director:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QComboBox" name="filmDirector_comboBox"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="movieGenre_label_2">
         <property name="text">
          <string>Genre:</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QComboBox" name="movieGenre_comboBox"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="movieCover_label_2">
         <property name="text">
          <string>Cover:</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QPushButton" name="uploadCover_button">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="movieCoverPath_label">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="8" column="1">
        <widget class="QPushButton" name="moviePreview_button">
         <property name="text">
          <string>Preview</string>
         </property>
        </widget>
       </item>
       <item row="9" column="0">
        <widget class="QLabel" name="movieDescription_label_2">
         <property name="text">
          <string>Description:</string>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QPlainTextEdit" name="movieDescription_plainTextEdit"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="movieStatus_label">
         <property name="text">
          <string>Status:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QComboBox" name="movieStatus_comboBox"/>
       </item>
       <item row="11" column="0" colspan="2">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="1">
        <widget class="QPushButton" name="editMovie_button">
         <property name="text">
          <string>Edit movie</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QPushButton" name="deleteMovie_button">
         <property name="text">
          <string>Delete movie</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
